### ISSUE TITLE
Support more options than dashicons for "images"

### ISSUE THREAD

**Issue by bobbingwide** [wp-shortcake/shortcake#121]
With hundreds of shortcodes possible in one WordPress installation, there may not be enough dashicons to choose from for the Insert Post Element List display. 

It would be nice if the dashicons solution could be extended to support genericons and other icon fonts.

Note: To display an actual image you just need to return an image tag.

`<img class="" src="http://qw/wp-content/plugins/oik/images/bbboing-icon-256x256.jpg" title="bbboing" alt="bbboing">`

To display one line help for the shortcode, just wrap the text in a paragraph tag.
`<p>one line help for shortcode</p>`

otherwise, you get very large text.

Currently it seems that the HTML is escaped. So it's not possible to use &copy; to display the Copyright symbol, for example.

**Comment by danielbachhuber** [comment]
I agree. I think our argument definition for shortcode UI needs revisiting.

**Comment by mattheu** [comment]
I think we should support a custom image, or dashicons. This should be pretty easy.

**Comment by cliffordp** [comment]
:+1:
